infra/bridge/src/hooks/twitter.js

```diff
@@ -0,0 +1,172 @@
+'use strict'
+
+const { OAuth } = require('oauth')
+const request = require('superagent')
+
+const logger = require('./../logger')
+
+const oauth = new OAuth(
+  'https://api.twitter.com/oauth/request_token',
+  'https://api.twitter.com/oauth/access_token',
+  process.env.TWITTER_CONSUMER_KEY,
+  process.env.TWITTER_CONSUMER_SECRET,
+  '1.0',
+  null,
+  'HMAC-SHA1'
+)
+
+const HOST =
+  process.env.NODE_ENV === 'development'
+    ? process.env.WEBHOOK_TUNNEL_HOST
+    : process.env.HOST
+
+const HOOK_ENV = process.env.TWITTER_WEBHOOK_ENV || 'dev'
+
+/**
+ * List all registered webhooks
+ * @param {String} oAuthToken
+ * @param {String} oAuthAccessTokenSecret
+ */
+function getWebhooks(oAuthToken, oAuthAccessTokenSecret) {
```

Comment:
should this be an async function since it returns a Promise. Same question for function createWebhook below.

Reply:
It doesn't have to be, unless we are using await inside this function. I have some inconsistencies in async/await/returning Promise though. Will clean it up.